### PR TITLE
[Naxx] Move Gluth to CombatAI

### PR DESCRIPTION
**Pull request by insunaa**
## 🍰 Pullrequest

I guess while I'm at it might as well move some easy bosses to CombatAI

### Proof

- None

### Issues

- None

### How2Test

- None

### Todo / Checklist

- [ ] Please check for logical consistency and test this at least once. I am currently unable to test things

### CODE REVIEW

**Comment by insunaa** [comment]
Ah, ok good thing I went over things again, I didn't delay timers when stunned by decimate...

**Comment by insunaa** [comment]
Not sure about the despawning thing. I mean they all do actually despawn on fail or done in the reference, but I'm not sure if this case isn't handled elsewhere already

File: src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/naxxramas/instance_naxxramas.cpp [hunk 1]
```diff
@@ -22,6 +22,7 @@ SDCategory: Naxxramas
 EndScriptData */
 
 #include "AI/ScriptDevAI/include/sc_common.h"
+#include "AI/ScriptDevAI/include/sc_instance.h"
```
Comment (insunaa) [inline src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/naxxramas/instance_naxxramas.cpp:25]: sorry about this bit... my dang IDE automatically adds includes it thinks it needs... this part can be omitted

**Comment by cala** [comment]
Merged in https://github.com/cmangos/mangos-tbc/commit/6e405063704813956810e9b9519ffc28e9d6bf94
Thank you !

**Pull request closed without merging**